### PR TITLE
Better start call setting

### PR DESCRIPTION
**Pull request by nickvergessen**
### Before
Setting wont save and the dropdown text is shorten
Bildschirmfoto von 2019-10-07 13-16-21

### After
All nice and shiny again (font size change is because of github image cropping)
Bildschirmfoto von 2019-10-07 13-15-57

### CODE REVIEW

**Comment by tigernero79** [comment]
when is the next update online?  thanks for everything.

File: vue/src/views/GeneralSettings.vue [hunk 1]
```diff
@@ -30,7 +30,11 @@
 				v-model="startCalls"
 				:options="startCallOptions"
 				label="label"
-				track-by="value" />
+				track-by="value"
+				@input="saveChanges" />
+		</p>
+		<p>
+			<em>{{ t('spreed', 'When a guest or user can not start a call they can still join the conversation and chat. When a user or moderator started the call they can also join the call.') }}</em>
```
Comment (Ivansss) [inline vue/src/views/GeneralSettings.vue]: The description is a bit confusing. I would say something like "When a call has started, everyone who has access to the conversation will be able to join the call."

Comment (nickvergessen) [reply vue/src/views/GeneralSettings.vue]: I wanted to make it clear that they can still chat, so people dont get it confused with the lobby thing

**Pull request merged**